pkg/apis/core/v1beta1/types_project.go

```diff
@@ -106,7 +106,7 @@ type ProjectMember struct {
 	// Role represents the role of this member.
 	// IMPORTANT: Be aware that this field will be removed in the `v1` version of this API in favor of the `roles`
 	// list.
-	// TODO: Remove this field in favor of the `owner` role in `v1`.
+	// TODO: Remove this field in favor of the `roles` list in `v1`..
```

Comment:
```suggestion
	// TODO: Remove this field in favor of the `roles` field in `v1`.
```

Reply:
I pushed a commit to fix the typo.  I also ran `make generate` to be sure there were no downstream changes needed.